nebullvm/operations/optimizations/compilers/quantizations/pytorch.py

```diff
@@ -151,13 +163,21 @@ def quantize_torch(
 
     if quantization_type is QuantizationType.HALF:
         input_tfms.append(HalfPrecisionTransformation())
-        return _half_precision(model), input_tfms
+        quantized_model = _half_precision(model)
     elif quantization_type is QuantizationType.STATIC:
-        return _quantize_static(model, input_data_torch, device), input_tfms
+        quantized_model, _ = (
+            _quantize_static(model, input_data_torch, device),
+            input_tfms,
+        )
     elif quantization_type is QuantizationType.DYNAMIC:
-        return _quantize_dynamic(model, input_data_torch, device), input_tfms
+        quantized_model, _ = (
+            _quantize_dynamic(model, input_data_torch, device),
+            input_tfms,
+        )
```

Comment:
```suggestion
        quantized_model = _quantize_dynamic(model, input_data_torch, device)
```

I think this should be the correct format :) (it should be applied everywhere)